### ISSUE TITLE
Add new language for country names

### ISSUE THREAD

**Issue by mariansedivy** [OSMNames/OSMNames#81]
Hello,

I would like to add Slovak and Czech language the same way that are now name_fr, name_en etc., so I can type Chorvatsko instead of Croatia in the search box and get correct results. I have modified `getLanguageName()` and `getAlternativesNames()` in functions.sql and all of their occurrences (I have added `name_sk` and `name_cs` into the parameters and modified the body of functions by duplicating the logic for `name_fr` parameter. I have also modified export.rst and processing.rst by adding these new columns everywhere where `name_fr` is. But when running the `docker-compose run --rm osmnames` it still fails on this error `
psql:01_polygons.sql:26: ERROR:  column "name_sk" does not exist
LINE 25: ...olygon AS r , getParentInfo(getLanguageName(name, name_sk, n...
`. In processing.rst I have modified create SQL for polygons table: `CREATE TABLE osm_polygon AS
 (SELECT     
      id,
      osm_id,
      type,
      country_code,
      name,
      name_sk,
      name_cz,
      name_fr,
      name_en,
      name_de,
      name_es,
      name_ru,
      name_zh,
      wikipedia...`. But the error is still there. Please can you help me with this?

Thank you very much.

**Comment by sfkeller** [comment]
Searching of all available names is already covered by indexing alternative_names; see http://osmnames.readthedocs.io/en/latest/architecture/export_data_format.html?highlight=alternative_names. 

An option to prefer which language goes into name is related to issue #55.

**Comment by mariansedivy** [comment]
Hi, I have read the docs and also the issue before, but both cover only the languages that are there by default. For Croatia it is `"alternative_names": "Хорватия,Hrvatska,Kroatien,Croacia,Croatie,克羅地亞"
`. But I need to have Chorvátsko, Chorvatsko also there. Or for Greece Grécko, Řecko. Now if I type Slovak names into the search box, I get wrong results.

![screen shot 2017-05-16 at 09 59 32](https://cloud.githubusercontent.com/assets/17254525/26095633/924c02e2-3a1e-11e7-9502-ca2a8f53b54a.png)
![screen shot 2017-05-16 at 09 59 44](https://cloud.githubusercontent.com/assets/17254525/26095635/926b2816-3a1e-11e7-8e6a-eb7a9fde9738.png)

**Comment by sfkeller** [comment]
Thanks for the hint. We have to check that. The problem can be at many places: 

1. in the OSM data, 
1. in the OSMNames source code (missing language, wrong ranking)
1. because older version of OSMNames export is installed at osmnames.org, 
1. because Sphinx search at osmnames.org is not optimally configured

**Comment by philippks** [comment]
osmnames.org does not yet use the newest version of the OSMNames export. When running osmnames-sphinxsearch locally with the newest export I get following results when typing your suggested search queries:
![image](https://cloud.githubusercontent.com/assets/8806985/26097593/65387a72-3a25-11e7-8123-9be3ff9610c1.png)

![image](https://cloud.githubusercontent.com/assets/8806985/26097639/98df2da8-3a25-11e7-8c89-c2fa3bb0fdf5.png)

**Comment by mariansedivy** [comment]
Great, this is exactly what I need. I will try this new version on my localhost. Thank you very much.

**Comment by mariansedivy** [comment]
Hello @philippks please can you tell me where did you get this version? I have cloned current master branch yesterday and installed it, but it is still without full alternative_names. Thank you.

**Comment by simran001** [comment]
Hi @mariansedivy,

It could be that osmnames-sphinxsearch is running with the default demo data (100k lines from an old planet export). So you could first get a new export (tsv file) from OSMNames (docker-compose run --rm osmnames). In this tsv file you will find the alternative_names in all available languages. Then you could run osmnames-sphinxsearch by specifying the path to the new tsv file as given [here](https://github.com/klokantech/osmnames-sphinxsearch#usage-of-docker-image). 

OSMNames by default does a full planet export, which currently takes quite long. @philippks has done some excellent performance optimization and will merge his changes with the master soon. But for now you could try with a smaller dataset, like Europe or a single country. You will find more information [here](https://github.com/OSMNames/OSMNames#get-started).

**Comment by philippks** [comment]
@mariansedivy Are the alternative_names missing in the exported file `export.tsv` after running `docker-compose run --rm osmnames`?

**Comment by mariansedivy** [comment]
Hi @simran001 and @philippks. I was doing it with Croatia dataset. Export.tsv contains all alternative names after running `docker-compose run --rm osmnames`. But the server was still running with the demo data. I have followed the Get started part from readme (https://github.com/OSMNames/OSMNames#get-started). After running the cut command and @philippks modified start command it works with the Croatia dataset. Now it works as expected. Thank you very much for your fast help.

For future generations:) Maybe this should be written in the docs, because I thought that changing `PBF_FILE_URL ` is all that has to be done. I have also missed the start command section that is in `osmnames-sphinxsearch` project, just something like "to start sample search server see [this](https://github.com/klokantech/osmnames-sphinxsearch#usage-of-docker-image)". Also `cut -f 1 --complement export.tsv > data.tsv` doesn't work on macOS, I had to run it in the docker container. 

This is a great project I really love it and wish you good luck with further development!
